lib/ui/form/widget/form_survey_answer_textarea.dart

```diff
@@ -1,17 +1,35 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_survey/model/answer_model.dart';
+import 'package:flutter_survey/model/submit_survey_question_model.dart';
 import 'package:flutter_survey/theme/app_dimensions.dart';
 
 class FormSurveyAnswerTextarea extends StatefulWidget {
-  const FormSurveyAnswerTextarea({super.key});
+  final ValueChanged<SubmitSurveyAnswerModel> onUpdateText;
+  final List<AnswerModel> answers;
+
+  const FormSurveyAnswerTextarea({
+    super.key,
+    required this.answers,
+    required this.onUpdateText,
+  });
 
   @override
   State<FormSurveyAnswerTextarea> createState() =>
       _FormSurveyAnswerTextareaState();
 }
 
 class _FormSurveyAnswerTextareaState extends State<FormSurveyAnswerTextarea> {
-  final _answerTextFieldController = TextEditingController();
+  @override
+  void initState() {
+    super.initState();
+    widget.onUpdateText(
+      SubmitSurveyAnswerModel(
+        id: widget.answers.first.id.toString(),
```

Comment:
This widget handles a single `answer` element. Should we send a single `answer` element instead?

```suggestion
        id: widget.answer.id.toString(),
```